src/plugins/chart_expressions/expression_partition_vis/public/__stories__/shared/arg_types.ts

```diff
@@ -54,13 +54,15 @@ export const argTypes: ArgTypes = {
   palette: {
     name: `${visConfigName}.palette`,
     description: 'Palette',
+    // @ts-expect-error update types
     type: { name: 'palette', required: false },
```

Comment:
With the required changes you can avoid the error silencing by: (I also changed palette to be required as it blows up storybook otherwise):
```suggestion
    type: { name: 'other', required: true, value: 'string' },
```